Jenkinsfile.cicd

```diff
@@ -83,6 +90,45 @@ pipeline {
       }
     }
 
+    stage('Tests') {
+      agent any
+      steps {
+        notifyStageStatus('Tests', 'PENDING')
+
+        script {
+          dir('frontend') {
+            try {
+              timeout(10) {
+              echo 'Installing NPM Dependencies...'
+              sh 'npm ci'
+
+              echo 'Linting and Testing Frontend...'
+              sh 'npm run test'
+
+              echo 'Frontend Lint Checks and Tests passed'
+
+              }
```

Comment:
There's missing indentation for lines 102-108 for the timeout contents, and an unnecessary whitespace line on 109.

Reply:
pushing fix